apps/package.json

```diff
@@ -228,6 +228,8 @@
     "@code-dot-org/js-interpreter": "^1.3.9",
     "@code-dot-org/snack-sdk": "2.2.0-apkurl-v1",
     "crypto-js": "^3.1.9-1",
+    "d3-geo": "^1.11.3",
+    "d3-scale": "^3.0.0",
```

Comment:
Looks like you didn't end up using d3-scale. Mind removing it?

Reply:
I was working on the next steps for this features which is to plot lat/long data points on the map which is where d3-scale comes in. I'll remove it when I end up not using d3-scale after plotting those points. Sorry about that, I should have remove d3-scale for this WIP PR to prevent confusion.